### PR TITLE
[JENKINS-67722] Fix `/configure` link on custom log recorders

### PR DESCRIPTION
**Pull request by NotMyFault**
Fixes [JENKINS-67722](https://issues.jenkins.io/browse/JENKINS-67722)

A regression from 2.322, where more tables have been modernized. The change proposed restores the original behavior, where clicking on the cogwheel of a custom log recorder redirects to `/log/<log recorder>/configure`, instead of /configure on the root URL.

### Proposed changelog entries

* The log recorder configure button links to the associated log recorder (regression in 2.322).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Comment by daniel-beck** [comment]
Could you file an issue so we can backport into 2.332.1?

**Comment by NotMyFault** [comment]
> Could you file an issue so we can backport into 2.332.x?

Added to the PR body 👍🏻

**Review by daniel-beck — APPROVED** [review]
Looks like a weird choice, but there's probably not a better one, and it just restores https://github.com/jenkinsci/jenkins/commit/0c5029392d99b6f1f7514f5ef1390eafec3751d7#diff-58a21354a8d8b1ac4db72b5a141485cd14f888acd5f32e5148db76779c753bafL62 (and is consistent with `local:row href` a few lines up.

**Review by timja — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
